### PR TITLE
New CLI help reference for studio and db commands

### PR DESCRIPTION
**Pull request by bholmesdev**
### new `astro --help` Studio Commands reference

image

### new `astro db --help` reference with matching formatting

image

## Testing

Manual call for the --help flag

## Docs

N/A

### CODE REVIEW

File: packages/db/src/core/cli/print-help.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+		for (const [tableTitle, tableRows] of tableEntries) {
+			message.push(linebreak(), title(tableTitle), table(tableRows, { padding }));
+		}
+	}
+
+	if (description) {
+		message.push(linebreak(), `${description}`);
+	}
+
+	// eslint-disable-next-line no-console
```
Comment (delucis) [inline packages/db/src/core/cli/print-help.ts:68]: According to the lint task this is unneeded:

```suggestion
```

File: .changeset/thirty-bags-live.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+"astro": patch
+"@astrojs/db": patch
+---
+
+Add --help reference for new db and studio commands
```
Comment (delucis) [inline .changeset/thirty-bags-live.md:6]: Changeset nit

```suggestion
Adds `--help` reference for new db and studio CLI commands
```

**Review by delucis — COMMENTED** [review]
Couple of notes but LGTM!

**Review by matthewp — APPROVED** [review]
Wonderful

**Pull request merged**